code/game/objects/structures/door_assembly.dm

```diff
@@ -329,7 +330,6 @@
 					new /obj/item/stack/sheet/glass(T)
 			else
 				new /obj/item/shard(T)
-		if(mineral)
-			var/obj/item/stack/sheet/mineral/mineral_path = text2path("/obj/item/stack/sheet/mineral/[mineral]")
-			new mineral_path(T, 2)
```

Comment:
With the removal of this string-based BS, does that mean the `mineral` var is deprecated, or is it still use for other things?

Reply:
Sadly it's still used in the examine

Reply:
It does get used quite often in the dm file.

My goal was at first to just fix the issue since this is my first PR.
I might refactor the code to remove its use along with the other text2path usages once I find the time to.

Reply:
I see, that's fine 👍

Reply:
@DevanStrife you could make `var/mineral_type` be `var/obj/mineral_type` then do `initial(mineral_type.name)` instead. however thats a much larger refactor that I am not gonna ask that you make